### PR TITLE
Add Android Makers 2018

### PR DESCRIPTION
**Pull request by benju69**
- **Conference Name**:
- **Conference URL**:
- **Why it should be added to `awesome-mobile-conferences`**:
- [x] Conference has a website
- [x] Conference has start/end date (if one day only conference write the same date)
- [x] It is an upcoming conference (no past conferences)
- [x] Check address using [Geocoder](https://google-developers.appspot.com/maps/documentation/utils/geocoder/) to get clean address
- [x] Updated **submission.json** instead of README or content.json
- [x] Put always your entry at the bottom of the array

```js
{
	"title": "MobOs 2018",
	"year": 2018,
	"startdate": "2018/02/15",
	"enddate": "2018/02/16",
	"where": "Cluj Napoca",
	"homepage": "http://romobos.com",
	"country": "Romania",
	"city": "Cluj",
	"callforpaper": false,
	"twitter": "@twitter",
    "topics": [ "android", "ios", "ui", "ux" ]
}
```

### CODE REVIEW

**Comment by matteocrippa** [comment]
@benju69 thank you for your PR, since yesterday I managed to modify the process to add new conferences, would you mind to add your entry in the `submission.json` file instead of the `content.json` one?

**Comment by benju69** [comment]
Yes ok sorry, I saw this after the PR

**Comment by matteocrippa** [comment]
Don't worry if you can update your PR it would be great.
I had to switch to this, because inside the submission only a few params are required, and then it generated dynamically both `content.json` and `Readme.md` adding extra fields useful for web and mobile version.

**Comment by benju69** [comment]
Should be ok now :)

File: submission.json [hunk 1]
```diff
@@ -899,7 +899,24 @@
...
+      "title": "Android Makers FR",
+      "year": 2018,
+      "startdate": "2018/04/23",
+      "enddate": "2018/04/24",
+      "where": "Le Beffroi 2 Place Emile Cresp, Montrouge",
+      "homepage": "http://androidmakers.fr/",
+      "country": "France",
+      "city": "Paris",
+      "callforpaper": true,
+      "emojiflag": "🇫🇷"
```
Comment (matteocrippa) [inline submission.json:913]: this line is missing a comma feel free to remove that, emoji flag is auto generated

File: submission.json [hunk 2]
```diff
@@ -899,7 +899,24 @@
...
+      "twitter": "@AndroidMakersFR",
+      "topics": [
+        "android"
+        ],
+      "emojiflag": "🇫🇷"
```
Comment (matteocrippa) [inline submission.json:918]: feel free to remove here too

File: submission.json [hunk 3]
```diff
@@ -899,7 +899,24 @@
...
+      "isnew": false
```
Comment (matteocrippa) [inline submission.json:919]: even is new is not needed here, it is also auto generated in contents

**Review by matteocrippa — COMMENTED** [review]
It seems that the json is invalid, can you check that?

**Pull request merged**